README.md

```diff
@@ -16,6 +16,10 @@ author:"hacktoberfest-team" type:"issues"
 
 (A good PR would be a search query which returns all repositories with the label "hacktoberfest" whilst checking the issues from "hacktoberfest-team")
 
+## Nice tip
+
+If you are struggling finding a good issue to contribute on, you can take a look at [Issuehub](http://issuehub.io). This page allows you to search for projects that have one or many specific labels you feel more comfortable with.
```

Comment:
Please make the changes as marked below so as to sound grammatically correct.

> If you are struggling ~~finding~~ _**to find**_ a good ~~issue~~ _**project**_ to contribute on

Since by working on an _issue_, a person contributes towards the _project_, hence the change.

> This page allows you to search for projects that have one or many specific labels you _**may**_ feel more comfortable with.

Reply:
Perfect! I will fix this right now!

Reply:
Amazing.